### ISSUE TITLE
Metric and imperial measuring options

### ISSUE THREAD

**Issue by 2fsbb9op** [0xf4b1/motionmate#80]
An option to default to km and cm (step width) or miles and inches would be welcomed in a future release.  Thanks for the consideration.

Happy New Year!

**Comment by 0xf4b1** [comment]
yeah true, I think I'll add this based on regional preferences in a new update
